src/Domain/Common/AuditableEntity.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
```

Comment:
We now have a `AuditEntity` and an `AuditableEntity`. Maybe `BaseEntity` would be a better name?

Reply:
One for the real entities, one for the audit. Maybe its too confusing